Formula/newt.rb

```diff
@@ -52,6 +50,8 @@ def install
 
   test do
     ENV["TERM"] = "xterm"
+    system "python3", "-c", "import snack"
```

Comment:
```suggestion
    system Formula["python@3.10"].opt_bin/"python3", "-c", "import snack"
```

Reply:
I'm not making this change as explained in https://github.com/Homebrew/homebrew-core/pull/95482#issuecomment-1046420238.

Reply:
I've made this change.